packages/frontend/src/components/DashboardTabs/index.tsx

```diff
@@ -232,122 +232,126 @@ const DashboardTabs: FC<DashboardTabsProps> = ({
-                                                    />
-                                                </TrackSection>
-                                            </div>
+                                            <DraggableTab
+                                                key={tab.uuid}
+                                                idx={idx}
+                                                tab={tab}
+                                                isEditMode={isEditMode}
+                                                sortedTabs={sortedTabs}
+                                                currentTabHasTiles={
+                                                    currentTabHasTiles
+                                                }
+                                                isActive={
+                                                    activeTab?.uuid === tab.uuid
+                                                }
+                                                setEditingTab={setEditingTab}
+                                                handleDeleteTab={
+                                                    handleDeleteTab
+                                                }
+                                                setDeletingTab={setDeletingTab}
+                                            />
                                         );
-                                    }
-                                })}
-                            </ResponsiveGridLayout>
-
+                                    })}
+                                    {provided.placeholder}
+                                    {isEditMode && (
+                                        <Group>
+                                            <Button
+                                                size="sm"
+                                                variant="outline"
+                                                leftIcon={
+                                                    <MantineIcon
+                                                        icon={IconPlus}
+                                                    />
+                                                }
+                                                onClick={() =>
+                                                    setAddingTab(true)
+                                                }
+                                                style={{
+                                                    borderWidth: 0,
+                                                }}
+                                            >
+                                                Add
+                                            </Button>
+                                        </Group>
+                                    )}
+                                </Tabs.List>
+                            )}
+                            <div style={{ padding: '0 1.25em 1.25em 1.25em' }}>
```

Comment:
Could we move the spacing to the `<ResponsiveGridLayout` component or the above component?

Reply:
I believe it breaks the ResponsiveGridLayout when added on that element, I remember trying that. I remember having a similar experience a long time ago using react-grid-layout, where it seems whatever spacing/padding calculations they're doing in there for organizing the grid items depends on there not being padding on the containing div. I also don't like adding an extra div here so I think I must have tried it on the above component as well but I can't remember. I can play around with it again and double-check.

Reply:
Current:
<img width="274" alt="image" src="https://github.com/user-attachments/assets/6aaf261a-e824-4ccc-9c38-965e9adffedf">


When placed on the closest above component (`<Tabs>`):
<img width="277" alt="image" src="https://github.com/user-attachments/assets/096deeb0-fd03-427b-a71c-435893981bc9">

When placed on `ResponsiveGridLayout`:
<img width="275" alt="image" src="https://github.com/user-attachments/assets/326a9d9f-d173-4920-b344-2ff0afe5db29">